cmd/crossbuild.go

```diff
@@ -104,19 +94,29 @@ func runCrossbuild() {
 	}
 
 	var (
-		goVersion        = viper.GetString("go")
-		repoPath         = viper.GetString("repository.path")
-		mainPlatforms    = viper.GetStringSlice("crossbuild.platforms.main")
-		ARMPlatforms     = viper.GetStringSlice("crossbuild.platforms.arm")
-		powerPCPlatforms = viper.GetStringSlice("crossbuild.platforms.powerpc")
-		MIPSPlatforms    = viper.GetStringSlice("crossbuild.platforms.mips")
+		mainPlatforms    []string
+		ARMPlatforms     []string
+		powerPCPlatforms []string
+		MIPSPlatforms    []string
+
+		goVersion = viper.GetString("go")
+		repoPath  = viper.GetString("repository.path")
+		platforms = viper.GetStringSlice("crossbuild.platforms")
 
 		dockerMainBuilderImage    = fmt.Sprintf("%s:%s-main", dockerBuilderImageName, goVersion)
 		dockerARMBuilderImage     = fmt.Sprintf("%s:%s-arm", dockerBuilderImageName, goVersion)
 		dockerPowerPCBuilderImage = fmt.Sprintf("%s:%s-powerpc", dockerBuilderImageName, goVersion)
 		dockerMIPSBuilderImage    = fmt.Sprintf("%s:%s-mips", dockerBuilderImageName, goVersion)
 	)
 
+	for _, platform := range platforms {
+		for platformGroup, platformGroupToBuild := range map[*[]string]*[]string{&defaultMainPlatforms: &mainPlatforms, &defaultARMPlatforms: &ARMPlatforms, &defaultPowerPCPlatforms: &powerPCPlatforms, &defaultMIPSPlatforms: &MIPSPlatforms} {
```

Comment:
I'd not pass string slice pointers, what is the reason for that? https://blog.golang.org/go-slices-usage-and-internals